src/core/retargeting_engine/python/retargeters/G1/trihand_motion_controller.py

```
@@ -115,13 +115,14 @@ def output_spec(self) -> RetargeterIOType:
             )
         }
 
-    def compute(self, inputs: RetargeterIO, outputs: RetargeterIO) -> None:
+    def _compute_fn(self, inputs: RetargeterIO, outputs: RetargeterIO, context) -> None:
```

Ruff (0.15.2)

[warning] Unused method argument: `context`

(ARG002)